charts/mirrorbits-parent/templates/_helpers.tpl

```diff
@@ -40,3 +40,70 @@ Must be 63 chars. maximum.
 {{- define "mirrorbits-parent.pv-secretname" -}}
 {{- printf "%s-%s" (include "mirrorbits-parent.name" . | trunc 43 ) "persistentvolume-secret" -}}
 {{- end -}}
+
+{{/*
+Define the azure file persistent volume claim name (if enabled).
+Must be 63 chars. maximum.
+*/}}
+{{- define "mirrorbits-parent.pvc-name" -}}
+{{- printf "%s-%s" .Release.Name "mirrorbits-parent-data" -}}
+{{- end -}}
+
+{{/*
+This template method checks the current ingress path and fail with a user facing message if needed or return empty.
+Expected argument: map{
+  "currentBackendService": <string>,
+  "rootContext": { },
+}
+ */}}
+{{- define "mirrorbits-parent.validateIngressBackend" -}}
+{{- if eq .currentBackendService "mirrorbits" -}}
+  {{- if not (index .rootContext.Values "mirrorbits-lite" "enabled") -}}
+    {{- fail "Cannot use mirrorbits(-lite) as backend if it is disabled." }}
```

Comment:
```suggestion
    {{- fail "Cannot use mirrorbits-lite as backend if it is disabled." }}
```
Will be far easier if/when we'll rename this chart to `mirrorbits` after deprecating the current/original `mirrorbits` chart.